static/src/jspm-config.js

```diff
@@ -0,0 +1,135 @@
+System.config({
+  "baseURL": "/assets",
+  "transpiler": "babel",
+  "paths": {
+    "*": "javascripts/projects/*.js",
+    "es6/*": "javascripts/es6/*.js",
+    "bootstraps/*": "javascripts/bootstraps/*.js",
```

Comment:
`jspm install` doesn't seem to touch this part of the config. If it wasn't safe to add paths here, how else could we add them? Any ideas?